pages/vercel-integration.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Layout,
+  Hero,
+  Wrapper,
+  Section,
+  RichTextWrapper,
+} from 'components/layout'
+import { NextSeo } from 'next-seo'
+import { LastModified } from './terms-of-service'
+import Link from 'next/link'
+
+export default () => (
+  <Layout>
+    <NextSeo title={'Vercel Integration'} description={'Vercel Integration'} />
+    <Hero>Vercel Integration</Hero>
+    <RichTextWrapper>
+      <Section>
+        <Wrapper>
+          <LastModified>Last Modified: August 9, 2021</LastModified>
+          Our integration lets you easily deploy your sites to Vercel through the Tina dashboard.
+          <br />
+          We will act on your behalf to create a project, configure environment variables, and trigger a deployment.
+          <br />
+          Once the project is created, feel free to revoke our integration's access, as it's only used for getting the project set up.
```

Comment:
but the next time you want to deploy a project you'll have to allow Tina too right?

Reply:
Yeah, as it is right now, it's creating a new Vercel integration installation every time you go through the flow, so then revoking access has no negative effect. I am currently working on using a different auth method that's recommended, which likely will have some implications regarding this. But generally speaking a user wouldn't be deploying the starter multiple times so it's safe to say that they can remove access since we don't continually need it.